config/dictionaries/os_version.json

```diff
@@ -0,0 +1,10 @@
+[
+  {
+    "id": "2.0",
```

Comment:
yeah, I mainly thought it would look weird in the URL, but you are probably right. I'll change it